WeWatch/WeWatch/Custom TabBar/CustomTabBar.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CustomTabBar.swift
+//  WeWatch
+//
+//  Created by Anton on 30/12/2024.
+//
+
+import SwiftUI
+
+internal struct CustomTabBar: View {
+    
+    @ObservedObject internal var viewModel: TabBarViewModel = .init()
```

Comment:
The `TabBarViewModel` shouldn’t be used here. Instead, you need a binding for the active `TabViewType`:
`@Binding private var activeTab: TabViewType`

Also, make sure to pass the value through the initializer.